.github/workflows/continous-integration-workflow.yaml

```diff
@@ -20,8 +20,14 @@ jobs:
       - name: Update apt repositories
         run: sudo apt update
 
-      - name: Install Java and Node
-        run: sudo apt-get install --yes default-jre default-jdk npm
+      - name: Install Java
```

Comment:
You don't need to do all this. Just replace `sudo apt-get install --yes default-jre default-jdk npm` with `sudo apt-get install --yes default-jre default-jdk nodejs` - same for the pipeline change